WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
First of all, it is not my pet project. I don't know where you got that from, but it is completely wrong. Frankly it becomes libel when you have no clue like this. 

> Frankly, players should be allowed to opt out of it by choosing it through a Rule. 

I'd love that to be the case: but it isn't possible to do it properly. Which is why the only solution is a sub-mod. 

> Your setup is ahistorical.

First of all, it isn't my setup. Secondly, using a system of mentalities to represent religious structures isn't ahistorical. Thirdly, you missed the fact that it is being changed based upon received feedback.